llvm/docs/CommandGuide/llvm-mc.rst

```diff
@@ -92,6 +92,11 @@ End-user Options
 
  Generate DWARF debugging info for assembly source files.
 
+.. option:: --hex
+
+ Take hex pairs as input for the disassembler.
+ Whitespace is ignored.
+
```

Comment:
It looks like "hexpair" is a term invented by radare/rizin and is not used anywhere else.
"hex pair" (with a space) is even less popular, google shows less than ten pages.
How about just "hex bytes"?

(I'm not nitpicking, I actually had to look at the implementation to understand what it means.)

Reply:
Thanks